packages/core/admin/admin/src/StrapiApp.js

```diff
@@ -23,6 +23,7 @@ import {
 import injectionZones from './injectionZones';
 import favicon from './favicon.png';
 import localStorageKey from './components/LanguageProvider/utils/localStorageKey';
+import fetchClient from './core/fetch-client';
```

Comment:
I think this is not a good location. Can we put this into a `utils` folder?

Reply:
ok, I agree 😃 Done